web/api/api.py

```diff
@@ -200,7 +200,7 @@ def update_plugin_category(slug, category):
     if not plugin:
         return api_util.api_not_found('No plugin with slug %s' % slug)
 
-    if not category in [c['id'] for c in get_all_categories_cached()]:
+    if category not in {c['id'] for c in get_all_categories_cached()}:
```

Comment:
This is actually a [set comprehension](https://docs.python.org/3/tutorial/datastructures.html#sets), not a [generator expression](https://www.python.org/dev/peps/pep-0289/). The generator expression version would be
```python
if category not in (c['id'] for c in get_all_categories_cached()):
```

This was a pretty off-topic comment and not a necessary change. I probably should have left it out, but I thought it was a good opportunity to mention a great feature of Python. Any of the three implementations should be functionally correct, so which one is used is mostly a style choice and definitely not a blocker on anything being merged.